docs/pages/editconfig.md

```diff
@@ -386,4 +386,17 @@ Parameters:
 * **ddb.table_name**: the name of the DynamoDB table where Dow Jones Hammer will put detection results. The default value is `hammer-rds-unencrypted`.
 * **accounts**: *optional* comma-separated list of accounts to check and report for issue in square brackets. Use this key to override accounts from **aws.accounts** in [config.json](#11-master-aws-account-settings);
 * **ignore_accounts**: *optional* comma-separated list of accounts to ignore during check. Use this key to exclude accounts from **aws.accounts** in [config.json](#11-master-aws-account-settings);
-* **reporting**: defines whether Dow Jones Hammer will report detected issues to JIRA/Slack. The default value is `false`;
\ No newline at end of file
+* **reporting**: defines whether Dow Jones Hammer will report detected issues to JIRA/Slack. The default value is `false`;
+
+### 2.13. Public AMI issues
+
+This section describes how to detect whether you have AMIs public access or not. Refer to [issue-specific playbook](playbook13_amis_public_access.html) for further details.
+
+Edit the **public_ami_issues** section of the `config.json` file to configure the handling of this issue.
+
+Parameters:
+* **enabled**: enables/disables issue identification. The default value is `true`;
```

Comment:
no `acounts`?

Reply:
These are the common parameters which we will use for any security issue. Accounts/Remediation accounts/ignore accounts sections are optional.